added July 2024 What's New (#510)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/introduction/whats-new.adoc

[[year-2024]]
== 2024

[[july-2024]]
=== July 2024

.Glossary entries

> * *xref:composite-content-view[composite content view]*: Updated glossary entry to lowercase spelling

Reviewer: I'm not sure if we should use "lowercase" as a verb, so maybe:

```suggestion
* *xref:composite-content-view[composite content view]*: Updated glossary entry to use lowercase spelling
```

> * *xref:content-view[content view]*: Updated glossary entry to lowercase spelling

Reviewer: ```suggestion
* *xref:content-view[content view]*: Updated glossary entry to use lowercase spelling
```

* *xref:nmstate[Nmstate]*: Added glossary entry

> * *xref:on-premise[on-premise]*: Updated glossary entry to include Red Hat OpenShift Container Platform definition

Reviewer: ```suggestion
* *xref:on-premise[on-premise]*: Updated glossary entry to include Red{nbsp}Hat OpenShift Container Platform definition
```

> * *Product*: Removed glossary entry

Reviewer: Can we add a "why" for these removed entries, so people know why they were removed? For example, I wouldn't know that "Product" would have been related to Satellite.

Maybe:

```suggestion
* *Product*: Removed glossary entry because Red{nbsp}Hat Satellite no longer capitalizes it
```

> * *Red{nbsp}Hat Network Proxy Server*: Removed glossary entry

Reviewer: ```suggestion
* *Red{nbsp}Hat Network Proxy Server*: Removed glossary entry because this term has been replaced by "Capsure Server"
```

> * *ed{nbsp}Hat Network Satellite Server*: Removed glossary entry

Reviewer: ```suggestion
* *Red{nbsp}Hat Network Satellite Server*: Removed glossary entry because this term has been replaced by "Satellite Server"
```

> * *Red{nbsp}Hat Network Proxy Server*: Removed glossary entry because this term has been replaced by "Capsure Server"

Reviewer: Apologies, I had a typo in my suggestion

```suggestion
* *Red{nbsp}Hat Network Proxy Server*: Removed glossary entry because this term has been replaced by "Capsule Server"
```

> * *xref:subscription-manifest[subscription manifest]*: Updated glossary entry to lowercase spelling

Reviewer: ```suggestion
* *xref:subscription-manifest[subscription manifest]*: Updated glossary entry to use lowercase spelling
```

.Style guidance

> * *xref:external-links[External links]*: Added guidance to avoid using URL shorteners.

Reviewer: I think we've been using no punctuation on these (though I do see some with them - probably could remove throughout any w/ them)

```suggestion
* *xref:external-links[External links]*: Added guidance to avoid using URL shorteners
```

> * *xref:shortdesc[Short descriptions]*: Added guidance to avoid starting modules with admonitions, including Technology Preview admonitions.

Reviewer: ```suggestion
* *xref:shortdesc[Short descriptions]*: Added guidance to avoid starting modules with admonitions, including Technology Preview admonitions
```
